fixes #44 : Improve Carousel

keraliss opened the pull request:
This is the design i came up with! 
![Screenshot from 2023-12-04 15-17-51](https://github.com/Bitshala/website-v2/assets/145189844/cdb376fa-f7fa-4364-8044-9cae0e809576)
![Screenshot from 2023-12-04 15-18-07](https://github.com/Bitshala/website-v2/assets/145189844/502f17a3-6142-4ed6-85e3-32225acec192)

emjshrx commented on src/components/common/Carousel.tsx:
```diff
@@ -1,76 +1,113 @@
+          <div className="my-5 lg:p-5 lg:text-3xl">
+            {slides[currentIndex].content}
+          </div>
+          <button className="mt-5 rounded-lg  border-2 border-dotted border-black  py-4 lg:mx-28">
```
Some hover effect?

emjshrx reviewed (commented):
Damn looks good 👌👌❤️❤️

keraliss commented on src/components/common/Carousel.tsx:
```diff
@@ -1,76 +1,113 @@
+          <div
+            key={slideIndex}
+            onClick={() => goToSlide(slideIndex)}
+            className="cursor-pointer p-2 text-2xl"
```
can you describe what kind of hover effect will be fitting?

keraliss commented:
added the col view 
![Screenshot from 2023-12-06 19-57-08](https://github.com/Bitshala/website-v2/assets/145189844/890a5bc5-eeb3-4ed0-8a3b-3330adfcff68)
also added emil to the team 
![Screenshot from 2023-12-06 20-00-52](https://github.com/Bitshala/website-v2/assets/145189844/b3cd3a99-5c11-4eb9-b502-4fc1054e6d96)

emjshrx commented on src/components/common/Carousel.jsx:
```diff
@@ -0,0 +1,149 @@
+import { useState } from "react";
```
Haha did you give up on Typescript already?

emjshrx commented on src/components/common/Carousel.jsx:
```diff
@@ -0,0 +1,149 @@
+        })}
+      </div>
+
+      <div className="group relative m-auto hidden w-full rounded-2xl bg-orange px-4 py-16  lg:block">
```
Can we split this and the earlier out into a separate components? in the same file itself? We can have two components and one parent component that hides and shows the correct carousel. 

Or maybe we can split out the card, since the card is same for both and then use that card within yhe desktop view and mobile. 

Just code refactor. This file looks too big and complicated with lots of repetition

emjshrx commented on src/pages/index.astro:
```diff
@@ -57,3 +57,4 @@ import Header from "../components/common/Header.astro";
   <People />
   <Footer />
 </Layout>
+ ../components/common/CarouselMobile.tsx
```
This is not needed.

keraliss replied:
was thinking of having a different component.

emjshrx commented:
Awesome work doing the mobile view so fast 👌👌. Also there are some merge conflicts due to my latest PR, please check them out
